data_safe_haven/config/pulumi.py

```diff
@@ -10,6 +10,7 @@
 class DSHPulumiProject(BaseModel, validate_assignment=True):
     """Container for DSH Pulumi Project persistent information"""
 
+    encrypted_key: str
```

Comment:
Again, need to update tests for this change.

Reply:
done in 1f0b05a840abac019e46bf44d8d404981f8c7a2d and 42a11c0164f66e40d89ad43de464ed4975a15d76